cmd/pebble/cmd_ls.go

```diff
@@ -16,24 +16,32 @@ package main
 
 import (
 	"fmt"
+	"os"
+	"text/tabwriter"
+
 	"github.com/jessevdk/go-flags"
 
 	"github.com/canonical/pebble/client"
 )
 
 type cmdLs struct {
 	clientMixin
-	Itself     bool `long:"itself"`
+	timeMixin
+
+	Directory  bool `short:"d" long:"directory"`
+	LongFormat bool `short:"l" long:"long"`
```

Comment:
Note that `ls` doesn't have the long version of this argument, so probably best to leave `--long` off.